frontend/express/public/stylesheets/main.css

```diff
@@ -851,6 +851,7 @@ div.datatablesubrow {display: none;}
 .cly-multi-select.selection-exists .default-text { display: none; }
 .cly-multi-select .default-text { display: block; padding: 5px 0; }
 .cly-multi-select .select-items .item.selected { color: #2fa732; display: none; }
+.cly-multi-select.disabled { opacity: 0.4 !important; cursor: default; }
```

Comment:
is this change related with nginx config issue?

Reply:
No. its related to the multi select disabled dropdowns.
Already merged to master.